code/index.html

```diff
@@ -1,30 +1,152 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Londrina+Solid:wght@100;300&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Dosis&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <title>Who's a good boy?</title>
+</head>
 
+<body>
+  <header>
+    <div class="hero">
+      <div class="hero-text">
+        <h1>Who's a good boy?</h1>
+        <p>If the answer is not your dog - even though you would never say that to their face - you might be in
+          need of some dog training. We believe that there's a good boy or girl slumbering in every dog.
+        </p>
+      </div>
+    </div>
+    <nav>
+      <img class="logo" src="./logo.png" alt="">
+      <a href="#">Our Dog Trainers</a>
+      <a href="#">Our Methods</a>
+      <a href="#">Our Customers</a>
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+    </nav>
+  </header>
+  <div class="container">
```

Comment:
Maybe this could be a `main` tag? An empty line between the header and the main content can be nice